src/rgw/rgw_admin.cc

```diff
@@ -489,6 +489,9 @@ void usage()
   cout << "\nradoslist options:\n";
   cout << "   --rgw-obj-fs                  the field separator that will separate the rados object name from the rgw object name;\n";
   cout << "                                 additionally rados objects for incomplete multipart uploads will not be output\n";
+  cout << "\nlist object in Bucket options:\n";
+  cout << "   --max-entries                 the number of objects will show one time, default 1000, no more than 10000\n";
```

Comment:
Perhaps write it as "max number of entries listed, default 1000, absolute maximum 10000".